src/features/dwn/profile-protocol/profile-protocol.ts

```diff
@@ -14,6 +18,13 @@ export const profileProtocol = {
           who: "anyone",
           can: "read",
         },
+        // TODO: remove the following rule.
+        // This only exists currently due to a bug where ManagedIdentities
+        // aren't being considered the DWN's tentant.
+        {
+          who: "anyone",
+          can: "write",
+        },
```

Comment:
Added a task to track this: https://github.com/TBD54566975/web5-wallet/issues/100